modules/op-running-a-pipeline.adoc

```diff
@@ -33,7 +33,7 @@ The <pipelinerun_id> in the above command is the ID for the `PipelineRun` that w
 +
 ----
 $ tkn pipeline start build-and-deploy \
-    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://github.com/openshift/pipelines-tutorial/blob/release-tech-preview-3/01_pipeline/03_persistent_volume_claim.yaml \
```

Comment:
You need to revert back to `raw.githubusercontent.com`, this URL would not work properly.